src/test/lib/parts_test.ts

```diff
@@ -124,6 +124,53 @@ suite('Parts', () => {
         assert.equal(stripExpressionMarkers(container.innerHTML), '');
       });
 
+      test('accepts a symbol', () => {
+        part.setValue(Symbol());
+        part.commit();
+        assert.equal(stripExpressionMarkers(container.innerHTML), 'Symbol()');
```

Comment:
To fix these tests with the Symbol polyfill, I think you want to toString the actual Symbol passed it:

```js
const symbol = Symbol();
part.setValue(symbol);
part.commit();
assert.equal(stripExpressionMarkers(container.innerHTML), String(symbol));
```